status/deliveries.py

```diff
@@ -3,10 +3,70 @@
 
 from status.util import SafeHandler
 
+from genologics.config import BASEURI, USERNAME, PASSWORD
+from genologics import lims
+from genologics.entities import Udfconfig
+lims = lims.Lims(BASEURI, USERNAME, PASSWORD)
 
 class DeliveriesPageHandler(SafeHandler):
-    def get(self):
 
+    def post(self):
+        project_id = self.get_argument('project_id', '')
+        responsible = self.get_argument('responsible', '')
+        if not project_id or not responsible:
+            self.set_status(400)
+            self.write('no project_id or bioinfo_responsible')
+            return
+        lims_projects = lims.get_projects()
```

Comment:
Getting all the projects in the lims is rather expensive. If you already have the project ID, it's faster to add 
`from genologics.entities import Udfconfig, Project`
and 
`lp = Project(lims, id=project_id)
lims_project = lp.name`